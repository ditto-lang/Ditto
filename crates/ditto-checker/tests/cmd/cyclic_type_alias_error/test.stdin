module Test exports (..)

type alias Foo = Foo

wut : Foo = unit
