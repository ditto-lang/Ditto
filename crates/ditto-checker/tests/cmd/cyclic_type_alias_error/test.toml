bin.name = "ditto-checker-testbin-check-module"
args = ["cyclic_type_alias_error.ditto"]
fs.sandbox = true
status = "failed"
