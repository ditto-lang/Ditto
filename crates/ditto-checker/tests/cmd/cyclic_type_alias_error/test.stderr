
  × types don't unify
   ╭─[cyclic_type_alias_error.ditto:4:1]
 4 │ 
 5 │ wut : Foo = unit
   ·             ──┬─
   ·               ╰── here
   ╰────
  help: expected Foo
        got Unit

